client/dashboard/task-list/tasks/payments/square.js

```diff
@@ -59,6 +65,13 @@ class Square extends Component {
 		);
 
 		try {
+			let newWindow = null;
+			if ( hasCbdIndustry ) {
+				// It's necessary to declare the new tab before the async call,
+				// otherwise, it won't be possible to open it.
+				newWindow = window.open( '/', '_blank' );
+			}
```

Comment:
Seems like we'd want to open this flow in a new window regardless of the CBD industry? (Most other payment gateway connection are occurring in new tabs/windows)

Reply:
Hi @jeffstieler,
As I understood from this [answer](https://github.com/woocommerce/woocommerce-admin/issues/4053#issuecomment-609608685), the new tab should only be opened on the CBD industry flow, not the normal Square flow.
Do you think it should be different?

Reply:
I don't feel too strongly about this, but it did strike me as odd that we aren't being consistent on behavior across all payment gateway connections.